kt-server panics, GetMutations fails with nil pointer 'exception'

HTTP-client receives a

`curl: (52) Empty reply from server` / `ERR_CONNECTION_CLOSED`

kt-server complains about:
```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x18 pc=0x84b772]

goroutine 47 [running]:
github.com/google/keytransparency/core/mutation.(*Server).lowestSequenceNumber(0xc4200e06e0, 0x7fd1c17021a8, 0xc42031c030, 0x0, 0x0, 0x1, 0x0, 0xc4202f8010, 0x0)
        /go/src/github.com/google/keytransparency/core/mutation/mutation.go:193 +0x2a2
github.com/google/keytransparency/core/mutation.(*Server).GetMutations(0xc4200e06e0, 0x7fd1c17021a8, 0xc42031c030, 0xc420436b70, 0xc4202f0500, 0xc420451878, 0x3)
        /go/src/github.com/google/keytransparency/core/mutation/mutation.go:76 +0x2cf
github.com/google/keytransparency/impl/mutation.(*Server).GetMutations(0xc4200dc090, 0x7fd1c17021a8, 0xc42031c030, 0xc420436b70, 0xc4200dc090, 0xad41bc, 0xc)
        /go/src/github.com/google/keytransparency/impl/mutation/mutation.go:41 +0x4c
github.com/google/keytransparency/impl/proto/mutation_v1_service._MutationService_GetMutations_Handler.func1(0x7fd1c17021a8, 0xc42031c030, 0xa5b460, 0xc420436b70, 0xc4203ae5a0, 0xc4203105a0, 0x0, 0xffd680)
        /go/src/github.com/google/keytransparency/impl/proto/mutation_v1_service/mutation_v1_service.pb.go:146 +0x86
github.com/google/keytransparency/vendor/github.com/grpc-ecosystem/go-grpc-prometheus.UnaryServerInterceptor(0x7fd1c17021a8, 0xc42031c030, 0xa5b460, 0xc420436b70, 0xc420310580, 0xc4203105a0, 0x0, 0xc4204519d8, 0x412d98, 0x50)
        /go/src/github.com/google/keytransparency/vendor/github.com/grpc-ecosystem/go-grpc-prometheus/server.go:29 +0xd2
github.com/google/keytransparency/impl/proto/mutation_v1_service._MutationService_GetMutations_Handler(0x9f6cc0, 0xc4200dc090, 0x7fd1c17021a8, 0xc42031c030, 0xc4203ae550, 0xaded50, 0x0, 0x0, 0x0, 0x0)
        /go/src/github.com/google/keytransparency/impl/proto/mutation_v1_service/mutation_v1_service.pb.go:148 +0x177
google.golang.org/grpc.(*Server).processUnaryRPC(0xc420198280, 0xfc9ac0, 0xc4204be180, 0xc4204f8200, 0xc4201a4d20, 0xfaac90, 0x0, 0x0, 0x0)
        /go/src/google.golang.org/grpc/server.go:781 +0xc41
google.golang.org/grpc.(*Server).handleStream(0xc420198280, 0xfc9ac0, 0xc4204be180, 0xc4204f8200, 0x0)
        /go/src/google.golang.org/grpc/server.go:981 +0x15a6
google.golang.org/grpc.(*Server).serveStreams.func1.1(0xc420338000, 0xc420198280, 0xfc9ac0, 0xc4204be180, 0xc4204f8200)
        /go/src/google.golang.org/grpc/server.go:551 +0xa9
created by google.golang.org/grpc.(*Server).serveStreams.func1
        /go/src/google.golang.org/grpc/server.go:552 +0xa1
```

--- a/core/mutation/mutation.go
+++ b/core/mutation/mutation.go
@@ -189,7 +189,7 @@ func (s *Server) lowestSequenceNumber(ctx context.Context, token string, epoch i
 			glog.Errorf("GetSignedMapRootByRevision(%v, %v): %v", s.mapID, epoch, err)
 			return 0, grpc.Errorf(codes.Internal, "Get previous signed map root failed")
 		}
-		lowestSeq = resp.GetMapRoot().GetMetadata().HighestFullyCompletedSeq
+		lowestSeq = resp.GetMapRoot().GetMetadata().GetHighestFullyCompletedSeq()
 	}
 	return uint64(lowestSeq), nil
 }

